src/pydvl/value/shapley/__init__.py

```diff
@@ -32,6 +38,8 @@ class ShapleyMode(str, Enum):
     CombinatorialMontecarlo = "combinatorial_montecarlo"
     PermutationMontecarlo = "permutation_montecarlo"
     TruncatedMontecarlo = "truncated_montecarlo"
+    Owen = "owen"
+    OwenAntithetic = "owen_anti"
```

Comment:
Minor nit: For the sake of consistency, I think `owen_anti` should be changed to `owen_antithetic`

Reply:
yikes, RSI says hello... But yes, I think you are right